### DISCUSSION TITLE
Added an Hyperbolic example to my Markdown Exec gallery

### DISCUSSION THREAD

**Discussion by pawamoy** [cduck/hyperbolic#34] [Show and tell]
(Sibling of https://github.com/cduck/drawsvg/discussions/131)

Markdown-Exec is an extension for Python-Markdown that allows to execute code blocks in Markdown pages 🙂

I added an example that shows how to generate an SVG drawing with Hyperbolic to my gallery of examples: https://pawamoy.github.io/markdown-exec/gallery/#with-hyperbolic. I couldn't find an example in the repo to get the colorful image. Feel free to paste something here or send a PR to make the gallery example even fancier!

**Comment by cduck** [comment]
Thanks for sharing!

The final examples in https://github.com/cduck/hyperbolic/blob/master/examples/isosceles-tiling.ipynb and https://github.com/cduck/hyperbolic/blob/master/examples/tiles.ipynb are more colorful if you'd like.
